cli/injector/cpu_pressure.go

```diff
@@ -18,10 +21,40 @@ var cpuPressureCmd = &cobra.Command{
 	PreRun: func(cmd *cobra.Command, args []string) {
 		// prepare spec
 		spec := v1beta1.CPUPressureSpec{}
+		podCores, err := getPodCores()
+		if err != nil {
+			log.Fatalw("error creating the node injector", "error", err)
+			return
+		}
+
+		cores, err := cpuset.Parse(podCores)
+		if err != nil {
+			log.Fatalw("error creating the node injector", "error", err)
+			return
+		}
 
+		log.Infow(fmt.Sprintf("target identified to be running on %d cores", cores.Size()), "cores", cores.ToSlice())
 		// create injector
 		for _, config := range configs {
-			injectors = append(injectors, injector.NewCPUPressureInjector(spec, injector.CPUPressureInjectorConfig{Config: config}))
+			injectors = append(injectors, injector.NewCPUPressureInjector(spec, injector.CPUPressureInjectorConfig{Config: config, DistinctPodCores: cores}))
 		}
 	},
 }
+
+func getPodCores() (string, error) {
+	var sb strings.Builder
+	for _, config := range configs {
+		log.Infow("retrieving target cpuset allocated cores")
```

Comment:
nit: I think we could also add some more logging information about what container it tries to retrieve the cpuset from, how much cpuset cores is there and use Debug to not generate too much noise? Or move this log line before the for loop to have a general log about the retrieving. Or both 🤷‍♀️ ?